data/json/snippets.json

```
@@ -1993,5 +1993,118 @@
     "type": "snippet",
     "category": "pyromania_withdrawal",
     "text": [ "You think of randomly lighting a fire, but decide against it." ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You think of steel blades and warm dripping blood." ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You'd like to hear the last breath of something living." ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "So much death around.  Why not add some more?" ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You lick your lips, in anticipation for dead trophies." ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "By the blade or by the gun? How will you kill this time?" ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "Ahh, how delightful would it be to kill something." ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You are death, and you are coming for them.  Soon." ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You whisper a song to yourself. \"Stab stab stab stab stab, and then you die!  Yeah!\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You whisper to yourself. \"Come, oh sweet death of yours.\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_withdrawal",
+    "text": [ "You whisper a song to yourself. \"I've seen the future, baby!  It is murder!\"" ]
+  },
+  {
+    "type": "snippet",
+    "category": "killer_on_kill",
+    "text": [ "You whisper to yourself. \"I have become death, the destroyer of worlds!\"" ]
```

Not sure if it's what you're going for, but the exact quote is
```suggestion
    "text": [ "You whisper to yourself. \"Now I am become death, the destroyer of worlds!\"" ]
```